src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="form form--big"
+    >
+
+      <label class="label">
+        <img
+          src="./images/Search.svg"
+          alt="icon"
+          class="icon"
+        >
+        <input
+          class="input input--big"
```

Comment:
```suggestion
      class="form"
    >

      <label class="label">
        <img
          src="./images/Search.svg"
          alt="icon"
          class="icon"
        >
        <input
          class="input"
```
lets make it work like this

Reply:
corrected